Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "xpx_supercontracts_sdk"
-version = "0.1.0"
+version = "0.2.0"
 authors = ["ProxymaX Core Development Team"]
 description = "ProximaX Supercontracts Rust SDK"
 edition = "2018"
```

Comment:
2020

Reply:
@alvin-reyes 
it's Rust edition. Exist only: 2015, 2018